overlays/bootstrap.nix

```diff
@@ -41,23 +41,23 @@ self: super: {
                 ++ self.lib.optional (versionOlder "8.6")                             ./patches/ghc/outputtable-assert-8.4.patch
                 ++ self.lib.optional (versionAtLeast "8.6" && versionOlder "8.6.4")   ./patches/ghc/MR148--T16104-GhcPlugins.patch
                 ++ self.lib.optional (versionOlder "8.6.4")                           ./patches/ghc/MR95--ghc-pkg-deadlock-fix.patch
+                ++ self.lib.optional (versionAtLeast "8.6" && versionOlder "8.8")     ./patches/ghc/iserv-proxy-cleanup.patch                             # https://gitlab.haskell.org/ghc/ghc/merge_requests/250  -- merged; ghc-8.8.1
```

Comment:
This one contains `remote-iserv`, as such disabling it, will remove `remote-iserv` :(

Guess we'll need to split that patch into two then? One for 8.6 and one for 8.8?